package ru.progwards.java2.lessons.tests.test.calc;

/*import org.junit.Rule;
import org.junit.Test;
import org.junit.rules.ExpectedException;
import ru.progwards.java2.lessons.tests.calc.SimpleCalculator;

 public class CheckExceptionMulTest {
     @Rule
     public ExpectedException exception = ExpectedException.none();

     @Test
     public void ExceptionWithMult() {
         exception.expect(RuntimeException.class);
         exception.expectMessage("Int range overflow during Mult " +
                 ((int) Integer.MAX_VALUE * 2));
         SimpleCalculator.mult(Integer.MAX_VALUE, 1);
     }
         @Test
         public void ExceptionWithMult1() {
             exception.expect(RuntimeException.class);
             exception.expectMessage("Int range overflow during Mult " +
                     ((int) Integer.MIN_VALUE * 2));
             SimpleCalculator.mult(Integer.MIN_VALUE, 1);
         }
     @Test
     public void ExceptionWithMult2() {
         exception.expect(RuntimeException.class);
         exception.expectMessage("Int range overflow during Mult " +
                 ((int) -2147483648 * 0));
         SimpleCalculator.mult(Integer.MIN_VALUE, 0);
     }
 }
       /* exception.expectMessage("Int range overflow during Mult " +
                ((int) Integer.MIN_VALUE <Integer.MIN_VALUE ));
        SimpleCalculator.mult(Integer.MIN_VALUE, 1);
    }
}*/


